src/chrome/content/rules/Redflagdeals.com.xml

```diff
@@ -0,0 +1,7 @@
+<ruleset name="Redflagdeals.com">
+	<target host="redflagdeals.com" />
+	<target host="forums.redflagdeals.com" />
+	<target host="www.redflagdeals.com" />
```

Comment:
See also:
```
backtoschool.redflagdeals.com
blackfriday.redflagdeals.com
creditcards.redflagdeals.com
boxingday.redflagdeals.com
feeds.redflagdeals.com
```
`www.redflagdeals.com` should be right under `redflagdeals.com`: https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#target-ordering